tests/test-sources/plugins/by-name/godot/default.nix

```diff
@@ -1,14 +1,21 @@
-{ pkgs, ... }:
-{
-  empty = {
+{ lib, pkgs, ... }:
+let
+  enable =
+    # TODO: 2025-04-20: build failure of godot_4
+    # https://github.com/NixOS/nixpkgs/issues/399818
+    # https://github.com/NixOS/nixpkgs/pull/400347
+    false
     # Godot is only available on Linux
-    plugins.godot.enable = pkgs.stdenv.isLinux;
+    && lib.meta.availableOn pkgs.stdenv.hostPlatform;
```

Comment:
This is a partially applied function. You need to pass the package as the second argument